api/HTMLElement.json

```diff
@@ -1493,10 +1493,12 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": "6"
+              "version_added": "16",
+              "version_removed": "49"
```

Comment:
Alright, with this change it's clear that we can remove the Microdata API from BCD, it's been gone from all browsers for a long enough. Not in this PR though.